ios/MusicControlManager.m

```diff
@@ -188,6 +188,14 @@ - (void) toggleHandler:(MPRemoteCommand *) command withSelector:(SEL) selector e
     command.enabled = enabled;
 }
 
+- (void) listenToAudioRouteChange:(BOOL) enabled {
```

Comment:
is this safe to do? I was worrying about the enable triggering multiple times for whatever reason (user error, or react lifecycle being triggered again) so I just followed the same approach you used for control toggles. If you think the else approach is still a good idea, and that this wouldn't keep adding listeners, I'm happy to make the change